apps/examples/linux_rpc_demo/linux_rpc_demod.c

```diff
@@ -223,8 +223,11 @@ int rpmsg_handle_term(void *data, struct rpmsg_rpc_svr *rpcs)
 	struct rpmsg_rpc_req_term *rpc_term_req = req_ptr;
 	struct rpmsg_endpoint *ept = &rpcs->ept;
```

Comment:
Looks like ept variable is redundant and can be removed.

Reply:
It is not needed in most of the functions in this demo, it is still checked in all of them anyway. The check doesn't make sense in the first place since this var is a pointer to a variable in `rpcs` which will never be 0 unless `rpsc` is 0, so we could just check for that. Like all the issues I've found in these demos, I'm simply ignoring all the ones that do not affect my refactoring work.

Reply:
If ept->name is  removed then "ept" and "rpcs" both are unused variable from this function. Can we remove them ? I agree with the change to remove ept->name.

Reply:
Yes, we can and should remove these from several spots in this example app. But that should be done in a follow up patch, this patch simply follows the standard that all other functions use (check for !ept even if not used).

Reply:
@arnopo , @edmooring are you okay with this?
Ideally if we should take care of unused variables in the same patch for, but if there is already follow up PR available, I am okay to ack this.

Reply:
@tnmysh 
Good ctach and your point seems to me valid. there's no need to reproduce the same bad code here. to fix it later
It's better to directly merge the good version.

Reply:
Fair enough, adding a patch to this series to fix these.

Reply:
> Fair enough, adding a patch to this series to fix these.

Thanks!
@tnmysh  ok for you with this?